src/components/Mission/MissionProfile.jsx

```diff
@@ -0,0 +1,25 @@
+import { useSelector } from 'react-redux';
+
+const MissionProfile = () => {
+  const { missionList } = useSelector((state) => state.missions);
+  const joinedMissions = missionList.filter((mission) => mission.reserved);
+  return (
+    <>
+      <h2>My Missions</h2>
+      {joinedMissions.length === 0 ? (
+        <p className="no-missions">No Missions Joined</p>
+      ) : (
```

Comment:
- I like how you are displaying `No missions Joined` when there are no joined missions. Great job :+1: